Wbooks/src/redux/navigationOptions/actions.js

```diff
@@ -0,0 +1,22 @@
+import { AsyncStorage } from 'react-native';
+
+export const actionTypes = {
+  LOG_OUT: 'LogOut',
+  SEARCH: 'Search'
+};
+
+const SESSION_DATA_KEY = 'sessionData';
+
+const navigationOptionsActions = {
+  logOut: navigation => async dispatch => {
+    await AsyncStorage.removeItem(SESSION_DATA_KEY).then(() => {
+      dispatch({
+        type: actionTypes.LOG_OUT
+      });
+      console.log('Data removed');
+      navigation.navigate('LogIn');
```

Comment:
I think this actions and its reducer are not necessary, you already have a `logOut` action on authActions that does the same except for the navigation.
Move navigation to logOut auth action, replace the call on the component that use this and delete this reducer and actions.

Reply:
same